audb/core/publish.py

```diff
@@ -117,7 +159,7 @@ def job(file):
     update_media = []
     audeer.run_tasks(
```

Comment:
I think it would be easier to understand if we could use a return value:
```python
update_media = audeer.run_tasks(
```
or would this not possible?
The same applies in `_put_media()`

Reply:
Ah, ok. Then we stick with the current solution.